vm/src/exceptions.rs

```diff
@@ -1182,6 +1182,20 @@ pub(super) mod types {
             )?;
             Ok(())
         }
+        #[pymethod(magic)]
+        fn reduce(exc: PyBaseExceptionRef, _vm: &VirtualMachine) -> PyTupleRef {
```

Comment:
```suggestion
        fn reduce(exc: PyBaseExceptionRef, vm: &VirtualMachine) -> PyTupleRef {
```
`_` means unused variable. But it is used below